.github/workflows/docker-build.yaml

```diff
@@ -0,0 +1,129 @@
+                uses: actions/download-artifact@v2
+                with:
+                    name: docker-compose
+            -   name: Build application
+                run: docker-compose up -d php-fpm
+            -   name: Check standards
+                run: docker-compose exec -T php-fpm php phing standards
+    tests-unit-functional-smoke:
+        name: Run unit, functional and smoke tests
+        needs: build-docker-images
+        runs-on: ubuntu-20.04
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref }}
+                uses: actions/checkout@v2
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Download docker-compose.yml from artifacts
+                uses: actions/download-artifact@v2
+                with:
+                    name: docker-compose
+            -   name: Build application
+                run: |
+                    docker-compose pull --parallel postgres elasticsearch redis php-fpm
+                    docker-compose up -d postgres elasticsearch redis php-fpm
+                    docker-compose exec -T php-fpm php phing db-create test-db-create db-demo elasticsearch-index-recreate elasticsearch-export error-pages-generate
+            -   name: Run tests
+                run: docker-compose exec -T php-fpm php phing tests
+            -   name: Upload logs to artifacts
+                if: ${{ failure() }}
+                uses: actions/upload-artifact@v2
+                with:
+                    name: tests-logs
+                    path: ./project-base/var/log/
+    tests-acceptance:
+        name: Run acceptance tests
+        needs: build-docker-images
+        runs-on: ubuntu-20.04
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref }}
+                uses: actions/checkout@v2
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Download docker-compose.yml from artifacts
+                uses: actions/download-artifact@v2
+                with:
+                    name: docker-compose
+            -   name: Build application
+                run: |
+                    docker-compose pull --parallel webserver postgres elasticsearch redis php-fpm selenium-server
+                    docker-compose up -d
+                    docker-compose exec -T php-fpm php phing dirs-create db-create test-db-create db-demo error-pages-generate test-db-demo test-elasticsearch-index-recreate test-elasticsearch-export tests-acceptance-build
+            -   name: Run acceptance tests
+                run: docker-compose exec -T php-fpm php phing tests-acceptance
+            -   name: PHP-FPM container logs
+                if: ${{ failure() }}
+                run: docker-compose logs php-fpm
+            -   name: Copy logs from container
+                if: ${{ failure() }}
+                run: docker cp shopsys-framework-php-fpm:/var/www/html/project-base/var/log ./project-base/var/log
```

Comment:
Logs are copied here from container, but it is missing in `tests-unit-functional-smoke` job. Also the previous job does not gathering container logs - is it intentional?